Add Yaml input and output formatters

andrewlock opened the pull request:
Addresses #103

damienbod commented:
cool, looks good, thanks @filipw

filipw commented:
wow brilliant - thanks @andrewlock. 
Can you push this out to Nuget, or do you need help?

I'm gonna merge it already, but can you also please add it to the Readme? it has a little [table of contents](https://github.com/WebApiContrib/WebAPIContrib.Core#formatters)

andrewlock commented:
Sure, no worries, thanks for adding it 😄 

I've just pushed to NuGet and added you as an owner. 

Updated ToC in fresh PR as this one already merged : #105
